images/linux/Ubuntu1804-README.md

```diff
@@ -35,7 +35,7 @@
 
 ### Project Management
 - Ant 1.10.5
-- Gradle 6.6.1
+- Gradle 
```

Comment:
something wrong with Gradle here

Reply:
Our regex (?<version>\d+\.\d+\.\d+) is incorrect, because Gradle is now 6.7. I will fix it.